torch/csrc/jit/export.cpp

```diff
@@ -27,48 +28,51 @@ struct ExportContext {
   onnx::OperatorExportTypes operator_export_type;
 };
 
-void encodeGraph(onnx::GraphProto * p_g, const std::shared_ptr<Graph> & g,
+void encodeGraph(onnx_torch::GraphProto * p_g, const std::shared_ptr<Graph> & g,
```

Comment:
What about `using onnx = onnx_torch`? That would be more readable, and would minimize the number of changes to this file.

Reply:
Best would be to use `ONNX_NAMESPACE` here so something like

```
void encodeGraph(ONNX_NAMESPACE::GraphProto * p_g, const std::shared_ptr<Graph> & g,
```

Reply:
Similar to @apaszke's thoughts, maybe you just want to do

```
namespace onnx_torch = ::torch::onnx; // instead of namespace onnx = ...
namespace onnx = ONNX_NAMESPACE;
```
above and just use that everywhere.